packages/core/template/portfolio/contents/experience.md

```diff
@@ -0,0 +1,9 @@
+<frontmatter>
+  title: Experience
+</frontmatter>
+
+<br>
+
+# Experience
+
+> This is a placeholder page for your work experiences
```

Comment:
Might be good to put the examples on using thumbnails and icons you have on the home page here instead!

Reply:
Thanks for the idea! I think placing the thumbnails and icons on the homepage should remain though because many of the examples have such thumbnails/icons on the homepage to give a quick overview of their skills. I think if we were to move the thumbnails/icons from the homepage, maybe I could move to the skills page instead?